project/FrontendBuild.scala

```diff
@@ -27,49 +27,42 @@ object FrontendBuild extends Build with MicroService {
 
   val compile = Seq(
-    "net.ceedubs"               %% "ficus"                    % ficus,
-    "uk.gov.hmrc"               %% "http-caching-client"      % cacheClient,
-    "uk.gov.hmrc"               %% "frontend-bootstrap"       % frontend,
-    "uk.gov.hmrc"               %% "play-config"              % playConfig,
-    "uk.gov.hmrc"               %% "play-json-logger"         % jsonLogger,
-    "uk.gov.hmrc"               %% "play-health"              % playHealth,
-    "uk.gov.hmrc"               %% "url-builder"              % urlBuilder,
-    "uk.gov.hmrc"               %% "play-whitelist-filter"    % playWhitelist,
-    "org.apache.httpcomponents" %  "httpclient"               % httpclient,
-    "com.mohiva"                %% "play-silhouette"          % silhouette
+    "com.iheart"                %% "ficus"                                    % "1.2.6",
+    "uk.gov.hmrc"               %% "http-caching-client"                      % "6.1.0",
+    "uk.gov.hmrc"               %% "frontend-bootstrap"                       % "7.10.0",
+    "uk.gov.hmrc"               %% "play-config"                              % "3.1.0",
+    "uk.gov.hmrc"               %% "logback-json-logger"                      % "3.1.0",
+    "uk.gov.hmrc"               %% "play-health"                              % "2.0.0",
+    "uk.gov.hmrc"               %% "play-whitelist-filter"                    % "2.0.0",
+    "uk.gov.hmrc"               %% "url-builder"                              % "1.0.0",
+    "org.apache.httpcomponents" %  "httpclient"                               % "4.5.3",
+    "org.apache.httpcomponents" %  "httpcore"                                 % "4.4.5",
+    "com.mohiva"                %% "play-silhouette"                          % silhouette,
+    "com.mohiva"                %% "play-silhouette-password-bcrypt"          % silhouette,
+    "com.mohiva"                %% "play-silhouette-crypto-jca"               % silhouette,
+    "com.mohiva"                %% "play-silhouette-persistence"              % silhouette,
+    "uk.gov.hmrc"               %% "play-filters"                             % "5.6.0",
+    "net.codingwell"            %% "scala-guice"                              % "4.1.0"
   )
 
   val test = Seq(
-    "org.scalatest"             %% "scalatest"                % scalatest     % "test",
-    "org.scalatestplus"         %% "play"                     % scalatestplus % "test",
-    "org.pegdown"               %  "pegdown"                  % pegdown       % "test",
-    "org.jsoup"                 %  "jsoup"                    % jsoup         % "test",
-    "com.github.tomakehurst"    %  "wiremock"                 % wiremock      % "test",
-    "uk.gov.hmrc"               %% "hmrctest"                 % hmrctest      % "test",
-    "com.mohiva"                %% "play-silhouette-testkit"  % silhouette    % "test"
+    "org.scalatest"             %% "scalatest"                    % "2.2.6"       % "test",
```

Comment:
We can remove this, the correct version of scalatest should be pulled in my scalatestplus.play